src/main/native/cpp/tcpsockets/TCPStream.cpp

```diff
@@ -55,7 +57,7 @@ TCPStream::TCPStream(int sd, sockaddr_in* address)
 
 TCPStream::~TCPStream() { close(); }
 
-std::size_t TCPStream::send(const char* buffer, std::size_t len, Error* err) {
+std::size_t TCPStream::send(const char* buffer, size_t len, Error* err) {
```

Comment:
Didn't remove std:: from return value.

Reply:
Funny you should mention that... there's a comment in test_stdlib.py: `# FIXME: Types at the beginning of the line are not matched.` :) It's because Python's regex engine doesn't handle using `^` in lookbehind assertions. I'll need to figure out another way to match on the beginning of a line.